app/views/users/show.html.erb

```diff
@@ -1 +1,44 @@
-<h1>Here is the profile page of user with id: <%= @id %></h1>
+<section class="showU">
+
+    <div class="oneuser">
+        <div class="photo">
+          <img src="<%= @user.photo %>" alt="<%= @user.name %> thumbnail" />
+        </div>
+        <div class="name">
+          <h2><%= @user.name %></h2>
+          <p>
+            Number of posts: <%= @user.posts_counter || 0 %>
+          </p>
+        </div>
+    </div>
+
+    <div class="bio">
+      <h3>Bio</h3>
+      <p>
+        <%= @user.bio %>
+      </p>
+    </div>
+
+    <div class="posts">
+      <% @user.posts.each do |post| %>
+        <div class="postL">
+          <h3><%= link_to post.title, user_post_path(@user, post) %></h3>
+          <p>
+            <%= post.text %>
+          </p>
+          <span class="like">
+            Likes: <%= post.likes.count || 0%>
+          </span>
+          <span class="comment">
+            Comments: <%= post.comments.count || 0 %>
+          </span>
+        </div>
+      <% end %>
```

Comment:
- [ ] Please use the `recent_posts` method to render the user's 3 most recent posts 🙏